.prettierignore

```diff
@@ -0,0 +1,2 @@
+# Ignore jinja2 variables within the cookiecutter project
+{{ cookiecutter.repo_name }}
```

Comment:
adding prettier but ignore `{{ cookiecutter.repo_name }}` folder;otherwise, it will modify jinja variables.